app/poros/food.rb

```diff
@@ -0,0 +1,12 @@
+class Food 
+  attr_reader :upc,
+              :description,
+              :brand_owner,
+              :ingredients
+  def initialize(data)
+    @upc = data[:gtinUpc]
```

Comment:
For UPC you could also add a check for if there is nothing returned. 

This is called a ternary and it might look like this:
```
@upc = data[:gtinUpc] ? data[:gtinUpc] : 'No response'
```
Or you could create a helper method that does the same as above like this:
```
def upc_format(data)
   if data[:gtinUpc]
        data[:gtinUpc]
   else
         'No response`
   end
end    
```
If you use to helper method, then in your initialize you would call this:
```
@upc = upc_format(data)      
```